plugins/dummy/factory.go

```diff
@@ -0,0 +1,26 @@
+package dummy
```

Comment:
I wouldn't name this `dummy`. It has negative connotations and doesn't really say much about what it is.
Maybe call it `noop` or `null`?